src/java.desktop/macosx/native/libawt_lwawt/awt/shaders.metal

```diff
@@ -0,0 +1,821 @@
+/*
+ * Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
looks like unrelated to the awt, move to the java2d.

Reply:
Moved under JDK-8260681